src/backgrounds.jl

```diff
@@ -0,0 +1,212 @@
+"""
+
+`draw_grid(video::Video, action::Action, frame::Int; grid_direction::Symbol = :TR)`
+
+Draws an oriented grid on the given frame of a Video.
+
+# Arguments
+- `grid_direction::Symbol`: Where grid animation finishes. Default: `:TR` Available Orientations:
+  - `:TR` - Animation finishes in the **T**op **R**ight corner of the frame.
+  - `:TL` - Animation finishes in the **T**op **L**eft corner of the frame.
+  - `:BR` - Animation finishes in the **B**ottom **R**ight corner of the frame.
+  - `:BL` - Animation finishes in the **B**ottom **L**eft corner of the frame.
+
+# Example
+This example will produce a grid animation that finishes in the top left corner of the animation. One will need to define their own path for `tempdirectory` and `pathname`.
+
+```
+using Javis
+using Luxor
+
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+video = Video(500, 500)
+javis(
+      video,
+      [
+          Action(1:100, ground),
+          Action(1:100, (args...) -> draw_grid(args..., grid_direction = :TL)),
+      ],
+      tempdirectory = "/tmp-directory",
+      creategif = true,
+      pathname = "/tmp-directory/grid_animation.gif"
+     )
+
+```
+
+"""
+function draw_grid(video::Video, action::Action, frame::Int; grid_direction::Symbol = :TR)
+
+    min_width = div(video.width, -2, RoundDown)
+    max_width = div(video.width, 2, RoundUp)
+
+    min_height = div(video.height, -2, RoundDown)
+    max_height = div(video.height, 2, RoundUp)
+
+    if grid_direction == :BR
+
+        for y_point = min_width:25:max_width
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
```

Comment:
You have this line at several places. I think it can be done before you have `if grid_direction`.

Reply:
Yup! I caught this in another part of the code but sadly did not apply it to all parts. :P